### PR TITLE
Add cypress tests for routable elements block

### PR DESCRIPTION
**Pull request by R0MANDEV**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Test block routable elements

### CODE REVIEW

File: web/rest/client/features/provider/conditionalRoutesCondition/putConditionalRoutesCondition.feature [hunk 1]
```diff
@@ -15,6 +15,7 @@ Feature: Update conditional routes conditions
           "routeType": "ivr",
           "numberValue": null,
           "friendValue": "",
+          "id": 1,
```
Comment (rbatistadev) [inline web/rest/client/features/provider/conditionalRoutesCondition/putConditionalRoutesCondition.feature:18]: you cannot put an element with an id in the body, you are currently updating /conditional_routes_conditions/1

File: web/portal/client/cypress/e2e/Friend/Friend.cy.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ describe('Friend', () => {
       rtpEncryption,
     });
 
-    cy.get('header').contains('Friends');
+    cy.contains('Friends');
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Friend/Friend.cy.js:79]:
```suggestion
    cy.get('header').contains('Friends');
```

File: web/portal/client/cypress/e2e/Friend/Friend.cy.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ describe('Friend', () => {
       'editFriend'
     );
 
-    cy.get('svg[data-testid="EditIcon"]').first().click();
+    cy.get('svg[data-testid="EditIcon"]').first().click({ force: true });
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Friend/Friend.cy.js:35]:
```suggestion
    cy.get('svg[data-testid="EditIcon"]').first().click();
```

File: web/portal/client/cypress/e2e/Friend/Friend.cy.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ describe('Friend', () => {
 
     cy.get('header').should('contain', 'Friends');
 
-    cy.get('table').should('contain', FriendCollection.body[0].name);
+    cy.get('table').should('contain', FriendCollection.body[0].id);
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Friend/Friend.cy.js:15]:
```suggestion
    cy.get('table').should('contain', FriendCollection.body[0].name);
```

File: web/portal/client/cypress/e2e/Friend/CallForwardSettings/CallForwardSettings.cy.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    cy.fillTheForm({
+      callTypeFilter,
+      callForwardType,
+      targetType,
+      numberValue,
+      friend,
+      extension,
+      voicemail,
+      enabled,
+    });
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Friend/CallForwardSettings/CallForwardSettings.cy.js:61]: In a separate function please, or if you prefer, set the other logic in this file, but do not create this kind of mixed logic

File: web/portal/client/cypress/e2e/Fax/FaxOut/FaxOut.cy.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
+import FaxInOutCollection from '../../../fixtures/FaxInOut/getCollection.json';
+import FaxInOutItem from '../../../fixtures/FaxInOut/getItem.json';
+import newFaxInOut from '../../../fixtures/FaxInOut/post.json';
+// import editFaxInOut from '../../fixtures/FaxInOut/put.json';
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Fax/FaxOut/FaxOut.cy.js:4]: remove this comment

File: web/portal/client/cypress/e2e/Fax/FaxOut/FaxOut.cy.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  ///////////////////////////////
+  // GET
+  ///////////////////////////////
+  it('get Fax Out Detail', () => {
+    cy.intercept('GET', '**/api/client/faxes_in_outs/1', {
+      ...FaxInOutItem,
+    }).as('getFax-1');
+
+    cy.get('svg[data-testid="PanoramaIcon"]').first().click();
+  });
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Fax/FaxOut/FaxOut.cy.js:57]: Why this?

File: web/portal/client/cypress/e2e/CallForwardSetting/CallForwardSetting.tests.js [hunk 8]
```diff
@@ -82,3 +82,44 @@ export const deleteCallForwardSetting = () => {
...
+    voicemail,
+    enabled,
+  });
+
+  cy.get('header').should('contain', 'Call forward settings');
+
+  cy.usePactWait('createCallForwardSettingToFriends')
+    .its('response.statusCode')
+    .should('eq', 201);
+};
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/CallForwardSetting/CallForwardSetting.tests.js:125]: Do not create this, use the existent one in the top of the file, actually is ignoring tons of attributes from newItem

File: web/portal/client/cypress/e2e/Fax/FaxIn/FaxIn.cy.js [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    cy.contains('Routing endpoints').click();
+    cy.contains('Faxes').click();
+
+    cy.get('header').should('contain', 'Faxes');
+
+    cy.get('svg[data-testid="OutboundIcon"]').eq(1).click();
+
+    cy.get('table').should('contain', FaxInOutCollection.body[0].id);
+  });
+
```
Comment (rbatistadev) [inline web/portal/client/cypress/e2e/Fax/FaxIn/FaxIn.cy.js:18]: You are not testing the creation of faxIn entity

**Review by rbatistadev — APPROVED** [review]

File: web/portal/client/cypress/fixtures/Friend/getCollection.json [hunk 10]
```diff
@@ -1,12 +1,14 @@
...
-      "id": 1,
-      "domain": 3
+        "name": "testFriend",
+        "description": "",
+        "priority": 1,
+        "directConnectivity": "yes",
+        "id": 1,
+        "company": 1,
+        "domain": 3,
+        "interCompany": null
```
Comment (Kaian) [inline web/portal/client/cypress/fixtures/Friend/getCollection.json:11]: This is not testing all existing Friend types. I would at least add interpbx friends to this list like API rest fixtures have: https://github.com/irontec/ivozprovider/blob/bleeding/web/rest/client/features/provider/friend/getFriend.feature#L15-L38

**Review by rbatistadev — APPROVED** [review]

**Pull request merged**